modules/monitoring-investigating-why-user-defined-metrics-are-unavailable.adoc

```diff
@@ -132,12 +132,7 @@ data:
 # ...
 ----
 +
-.. Save the file to apply the changes.
-+
-[NOTE]
-====
-The `prometheus-operator` in the `openshift-user-workload-monitoring` project restarts automatically when you apply the log-level change.
-====
+.. Save the file to apply the changes. The new configuration is applied automatically.
```

Comment:
no need to change
apply debug loglevel for `prometheus-operator`, `prometheus-operator` pod would recreate
```
$ oc -n openshift-user-workload-monitoring get pod
NAME                                   READY   STATUS    RESTARTS   AGE
alertmanager-user-workload-0           6/6     Running   0          178m
alertmanager-user-workload-1           6/6     Running   0          178m
prometheus-operator-6c6d7d75bc-knbgb   2/2     Running   0          31s
prometheus-user-workload-0             6/6     Running   0          3h11m
prometheus-user-workload-1             6/6     Running   0          3h11m
thanos-ruler-user-workload-0           4/4     Running   0          178m
thanos-ruler-user-workload-1           4/4     Running   0          178m

$ oc -n openshift-user-workload-monitoring get pod prometheus-operator-6c6d7d75bc-knbgb -oyaml | grep log-level
    - --log-level=debug
```
maybe we can change the original note to
The `prometheus-operator` pod in the `openshift-user-workload-monitoring` project restarts automatically when you apply the log-level change.

just add `pod`

Reply:
Thank you, addressed. I changed it to:
```
The affected `prometheus-operator` pod is automatically redeployed.
```
to be consistent with the rest of the docs :+1: